méli-mélo/2024-04-stepsquiz/stepsquiz-limited.html

```diff
@@ -0,0 +1,126 @@
+---
```

Comment:
Apply the same changes as requested for the other web page. Also inter-link them too.